File: src/main/java/com/lawencon/elearn/dto/quizschedule/QuizScheduleInsertResDto.java
package com.lawencon.elearn.dto.quizschedule;

public class QuizScheduleInsertResDto {
	private QuizScheduleInsertDataResDto data;
	private String message;
	public QuizScheduleInsertDataResDto getData() {
		return data;
	}
	public void setData(QuizScheduleInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}

File: frontend/src/app/dto/quiz-schedule/quiz-schedule.insert.res.dto.ts
import { QuizScheduleInsertDataResDto } from "./quiz-schedule.insert.data.res.dto";

export class QuizScheduleInsertResDto{
    private _data! : QuizScheduleInsertDataResDto
	private _message! : string

    public get data(): QuizScheduleInsertDataResDto {
        return this._data;
    }

    public set data(data: QuizScheduleInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}